_pages/model-deployment/xgboost.md

```diff
@@ -140,8 +149,8 @@ def process_input(input):
             print("Could not create numpy array from data", e)
             sys.exit(0)
 
-# API calls will begin at the apply() method, with the request body passed as 'input'
-# For more details, see {{site.url}}{{site.baseurl}}/algorithm-development/languages
+#API calls will begin at the apply() method, with the request body passed as 'input'
```

Comment:
Why remove space here </nit>

Reply:
No reason, that was apparently my markdown editor's bad suggestion. Fixed with the latest commit.